api/src/test/java/org/apache/iceberg/expressions/TestInclusiveManifestEvaluator.java

```diff
@@ -753,4 +769,167 @@ public void testIntegerNotIn() {
         ManifestEvaluator.forRowFilter(notIn("no_nulls", "abc", "def"), SPEC, true).eval(FILE);
     assertThat(shouldRead).as("Should read: notIn on no nulls column").isTrue();
   }
+
+  /** Tests UUID equality filter using byte-order comparison against partition bounds. */
```

Comment:
nit: I believe all of these comments can be removed, since it's obvious from the method naming